test/setup/versions.tf

```diff
@@ -19,9 +19,9 @@ terraform {
 }
 
 provider "google" {
-  version = "~> 2.13.0"
+  version = "~> 2.12.0"
```

Comment:
any tests done with `2.20.1` ? I had some IAM permissions issues when testing with `2.13.0` and above. This is not necessary to be tested right now, just curious.

Reply:
I haven't tested with anything higher than 2.13. I know we've had a lot of modules start breaking because of an upstream issue at 2.13 and the solution we've been using was to pin to 2.12.